CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ include(cmake/set_version_numbers.cmake)
 set(${PROJECT_NAME}_CXX_FLAGS "${${PROJECT_NAME}_CXX_FLAGS} -DFUSION_MAX_MAP_SIZE=30 -DFUSION_MAX_VECTOR_SIZE=30 -D_LIBCPP_ENABLE_CXX17_REMOVED_AUTO_PTR")
```

Comment:
As a side note: We can remove -`DFUSION_MAX_MAP_SIZE` and `-DFUSION_MAX_VECTOR_SIZE` here, they are no longer necessary with C++11 and higher. (Otherwise they would need to be exported as public)

Reply:
they are actually exported, see line 101

Reply:
I'm removing them. Just testing that everything compiles downstream